packages/psss/src/api/queries/useSingleWeeklyReport.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { queryCache } from 'react-query';
+import { useData } from './useData';
+
+const getAlerts = row =>
+  Object.entries(row).reduce(
+    (list, [key, value]) =>
+      key.includes('Threshold Crossed') && value
+        ? [...list, key.split('Threshold Crossed')[0]]
+        : list,
+    [],
+  );
+
+const getUnVerifiedAlerts = (alerts, verifiedStatuses) =>
+  alerts.reduce(
+    (list, syndrome) => (verifiedStatuses.includes(syndrome.id) ? list : [...list, syndrome.id]),
+    [],
+  );
```

Comment:
Equivalent, but probably simpler:

```suggestion
  alerts.filter(alert => !verifiedStatuses.includes(alert.id));
```

Reply:
Yes agreed. It assumes that the alerts is a simple array of strings which it obviously is. I've added some comments to make that all more obvious and added this file to my card for unit testing.